lighthouse-core/runner.js

```diff
@@ -25,6 +25,32 @@ const fs = require('fs');
 const path = require('path');
 const url = require('url');
 
+function runAudit(audit, artifacts) {
+  const status = `Evaluating: ${audit.meta.description}`;
+
+  return Promise.resolve().then(_ => {
+    log.log('status', status);
+
+    // Return an early error if an artifact required for the audit is missing.
+    for (const artifactName of audit.meta.requiredArtifacts) {
+      // TODO: need pass-specific check of networkRecords and traces
```

Comment:
as discussed in person, since we don't even check that `networkRecords['passName']` exists, the simpler check for just `traces[Audit.DEFAULT_PASS]` plus a TODO will be sufficient, since we'll need to clean up handling of pass names anyway